Removed hardcoded social media and mobile store Images hosted on Sailthru

juakali-networks opened the pull request:
Removed links to Images that were hosted on Sailthru from the base edx template below.

 https://github.com/openedx/edx-platform/blob/db32ff2cdf678fa8edd12c9da76a76eef0478614/openedx/core/djangoapps/ace_common/templates/ace_common/edx_ace/common/base_body.html

Replaced the image sources to the social medial and mobile store links in base template by a variable. The URL links and the logos for the social media and mobile store links are now defined in the lms/envs/common.py file.

This fixes this issue https://github.com/openedx/build-test-release-wg/issues/148

juakali-networks committed:
fix: Removed hardcoded social media and mobile store images.

openedx-webhooks commented:
Thanks for the pull request, @Peters-Lab!  Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.




<!-- comment:external_pr -->
<!-- jenkins ok to test -->
<!-- data: eyJkcmFmdCI6IGZhbHNlfQ== -->

natabene commented:
@Peters-Lab Thank you for your contribution. Once you submit your CLA forms and they have been processed, we will be able to review your code.

juakali-networks commented:
@natabene I have finished submitting my CLA forms. Thanks

natabene commented:
@Peters-Lab Thank you for submitting the form, sorry it took so long. The CLA check is green now, I have kicked off other tests. Let's see how they turn out.

juakali-networks commented:
@natabene
I see the commitlint job failed with commit message errors.

Do I need to fix them?. Is any action required of me?.

natabene commented:
@Peters-Lab Ideally, yes, it would be good to fix them.

natabene commented:
@wajeeha-khalid FYI this PR needs review and merge.

attiyaIshaque commented:
@Peters-Lab Can you please look into failed unit-tests? https://github.com/openedx/edx-platform/runs/7434652162?check_suite_focus=true

juakali-networks commented:
@attiyaIshaque 
@natabene 

OK. I will look at the failing unit tests and get back to you.

attiyaIshaque commented:
Hi @Peters-Lab, Any update on this?
Thank you.

juakali-networks commented:
Hi @attiyaIshaque 
I have no updates yet. I got carried away by some personal stuff but am now free. I am looking at the issue now.

juakali-networks commented:
Hi @attiyaIshaque 

I have fixed the issue that was possibly causing the unit tests to fail. I have updated the PR.
Could you start the tests again?

natabene commented:
@Peters-Lab I kicked off the tests.

juakali-networks commented:
@natabene 
Thanks Natalia.

I still see commitlint errors in the result. Will try to understand them and possibly fix them tomorrow.

attiyaIshaque commented:
@Peters-Lab you can fix the commit lint message by updating like this :
`fix: Removed hardcoded social media and mobile store Images hosted on Sailthru`

Also this tests failed:

[AttributeError: 'Settings' object has no attribute 'MOBILE_STORE_URLS'](https://github.com/openedx/edx-platform/runs/7655595595?check_suite_focus=true#step:7:87507)

Can you please look into this?

juakali-networks commented:
@attiyaIshaque 

OK. Thanks. Let me have a look at the failing test.

juakali-networks commented:
Hi @attiyaIshaque,
I have renamed the PR, hope that fixes the commitlint errors. I have a fix also for the issue that was causing the AttributeError. I have updated the PR. The tests will tell us if all is ok, or if I need to make any additional changes.

Hi @natabene,
 could you start the tests?

attiyaIshaque commented:
@Peters-Lab Can you please use this syntax for commit message ?

`fix: Removed hardcoded social media and mobile store Images hosted on Sailthru`

juakali-networks commented:
@attiyaIshaque I have used the syntax you sent as my commit message.

attiyaIshaque commented:
@Peters-Lab Can you please further look into failing tests?

juakali-networks commented:
@attiyaIshaque OK. I will have a look at them.

juakali-networks commented:
@attiyaIshaque 
The Social media URLs had been removed from the test_api.py framework. Possibly to make the tests run. I will add the URLs to test_api.py, though this should be reviewed.

juakali-networks commented:
@attiyaIshaque 
@natabene 
I have pushed my changes that fix the missing Social URLs in the test_api. You can start the tests.

natabene commented:
@Peters-Lab I kicked off the tests.

juakali-networks commented:
@natabene Thanks.

Am already fixing the issue that failed the tests.

juakali-networks commented:
@natabene 

I have pushed my fixes. Could you start the tests again?

attiyaIshaque commented:
`lms/djangoapps/branding/tests/test_api.py:150:0: C0301: Line too long (121/120) (line-too-long)
openedx/core/djangoapps/schedules/tests/test_resolvers.py:177:0: C0301: Line too long (121/120) (line-too-long)
openedx/core/djangoapps/schedules/tests/test_resolvers.py:278:0: C0301: Line too long (121/120) (line-too-long`


@Peters-Lab Can you please fix this and update the commit message

juakali-networks commented:
@attiyaIshaque 

Pylint returns the error that the URLs are too long. I will use shorter strings as URL variables. I will try to work on this tomorrow. 

As for the commit message, I tried in a previous test run to use the commit message you sent me but it didn't help. Should I also add an input and subject to the commit message?

attiyaIshaque commented:
Yes you should add to the commit message.

juakali-networks commented:
@attiyaIshaque 
I will use it as the commit message when I commit my fix for the pylint line to long errors.

juakali-networks commented:
@attiyaIshaque 
@natabene 
I have pushed the potential fix for the pylint long lines errors and the commit message. Could you please start the tests?.

juakali-networks commented:
@natabene 
Please give the tests another run. I've just fixed the whitespaces so one of the failing tests today should pass.

@attiyaIshaque will help me with the commit message as one of the tests is failing regardless of the commit message I enter

attiyaIshaque commented:
@Peters-Lab I have fixed the commit message issue and squash the commits. 
please see the final changes so i can merge it. 
Thanks

juakali-networks commented:
@attiyaIshaque 
I've had a look through and the changes are fine by me.

Thanks

openedx-webhooks commented:
@Peters-Lab 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/edx-platform&entry.867055334=https://github.com/openedx/edx-platform/pull/30630&entry.1484655318=https://github.com/Peters-Lab&entry.752974735=2022-06-21+17:03&entry.1917517419=2022-08-17+12:51&entry.2133058324=Yes) so we can improve your experience in the future.

<!-- comment:end_survey -->

attiyaIshaque commented:
@Peters-Lab changes are merged. thanks for the contribution 😄
